src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-create/rgw-storage-class-create.component.html

```diff
@@ -0,0 +1,358 @@
+          <span
+            class="invalid-feedback"
+            *ngIf="storageClassForm.showError('zonegroup', formDir, 'required')"
+            i18n
+            >This field is required.</span
+          >
+        </ng-template>
+      </div>
+      <div cdsCol>
+        <!-- Placement Target -->
+        <cds-select
+          label="Placement Target"
+          formControlName="placement_target"
+          id="placement_target"
+          [invalid]="
+            storageClassForm.controls.placement_target.invalid &&
+            storageClassForm.controls.placement_target.dirty
+          "
+          [invalidText]="placementError"
+          i18n-helperText
+        >
+        <option [value]=""
+                 i18n> --Select-- </option>
+        <option *ngFor="let placementTarget of placementTargets"
+                  [value]="placementTarget"
+                  [selected]="placementTarget === storageClassForm.getValue('placement_target')"
+                  i18n>
+            {{ placementTarget }}
+        </option>
+        </cds-select>
+        <ng-template #placementError>
+          <span
+            class="invalid-feedback"
+            *ngIf="storageClassForm.showError('placement_target', formDir, 'required')"
+            i18n
+            >This field is required.</span
+          >
+        </ng-template>
+      </div>
+    </div>
+    <!-- Storage Class -->
+    <div class="form-item">
+      <cds-text-label
+        labelInputID="storage_class"
+        i18n
+        [invalid]="
+          storageClassForm.controls.storage_class.invalid &&
+          storageClassForm.controls.storage_class.dirty
+        "
+        [invalidText]="storageError"
+        >Storage Class Name
+        <input
+          cdsText
+          type="type"
+          placeholder="Storage Class Name..."
+          i18n-placeholder
+          id="storage_class"
+          formControlName="storage_class"
+          [invalid]="
+            storageClassForm.controls.storage_class.invalid &&
+            storageClassForm.controls.storage_class.dirty
+          "
+        />
+      </cds-text-label>
+      <ng-template #storageError>
+        <span
+          class="invalid-feedback"
+          *ngIf="storageClassForm.showError('storage_class', formDir, 'required')"
+          i18n
+          >This field is required.</span
+        >
+      </ng-template>
+    </div>
+    <div class="form-item form-item-append"
+         cdsRow>
+      <div cdsCol>
+        <!-- Target Region -->
+        <cds-text-label
+          labelInputID="region"
+          i18n
+          [invalid]="
+            storageClassForm.controls.region.invalid && storageClassForm.controls.region.dirty
+          "
+          [invalidText]="regionError"
+          [helperText]="targetRegionText"
+          >Target Region
+          <input
+            cdsText
+            type="text"
+            placeholder="Target Region..."
+            i18n-placeholder
```

Comment:
```suggestion

```
Same here, we should try adding default values rather than placeholder if possible.
The placeholder if smilar to field label then lets omit. Thanks